pages/garages.html

```diff
@@ -31,8 +31,8 @@
 recommendation-3-link: https://www.radiotopia.fm/podcasts/the-memory-palace
 partner-link: dovetail
 partner-title: PRX Technology Lets Stations Take Control
-creator-link: keyshawn
-creator-title: Keyshawn Solves it
+creator-link: nairobi-popup
+creator-title: Nairobi Pop-up Garage
```

Comment:
This fix used a different title in PR #20. Either this PR needs to not include the fix, or gets merge before #20.

Reply:
Probably address this when handling the merge conflicts.

Reply:
Yes, resolving the merge conflicts fixed this.